packages/app/src/modals-manager/constants/modal-layers.enum.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export enum ModalLayers {
```

Comment:
🟡  I think that layers names should represent the system domain/feature or even part domain/feature eg. Default = 5, PrivacyPolicy, MCForceUpdace